data/pages-content/volunteers.md

```diff
@@ -1,48 +1,53 @@
 
-In return, we will have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.
+We will also have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.
 
 
-# Things to remember
-  - To enter the conference you need a ticket
-  - Yo can sign up to any tasks on our Volunteer App before you have bought a ticket
-  - You will not earn a ticket for being a volunteer
+## Things to remember
 
-# How to sign up
-  - Fill this form and we will contact and onboard you. https://forms.gle/Y472nH5m8ZRWpbVP8
-  - If you want to save some time you can:
-      + Acknowledge our [EuroPython 2022 Data Privacy Policy](https://docs.google.com/forms/d/e/1FAIpQLSdNTLsFn-YKD4l6icY4NEhmVG5JRyawm-QcvZUQ9HSylEXldw/viewform). This is needed, since many of you will have to handle private data as part of e.g. helping at the registration desk.
-      + Assign yourself to available shifts on the Operations spreadsheet. (You will get the link after signing the Data Privacy Policy.)
-      + Sign up to the Europython Volunteers discord server. (The link is in our Operations Spreadsheet)
-      + Please say hello and introduce yourself in #onsite-volunteers channel. Thanks ❤️
-  - If you have any problem with the process please email volunteers@europython.eu or find a volunteer fellow in the venue.
+  - To enter the conference you need a ticket.
+  - You can sign up to volunteer before having bought a ticket.
 
-# Perks for volunteers
 
-  - Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-) (1 slot activity)
-  - A special group photo with all volunteers (1 slot activity; time will be announced on the #onsite-volunteers)
+## Perks for volunteers
 
+  - A free conference ticket if you volunteer for eight hours or more!
+  - Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-)
+  - A special group photo with all volunteers (time will be announced on the #onsite-volunteers)
 
-# Summary of tasks
+
+## What Volunteering at EuroPython looks like
+
+As a part of EuroPython volunteers' fleet, you will be the face of the conference. Your prime directive will be to ensure everyone coming to the conference has a great time. We want volunteers to be welcoming, helpful, collaborative, and overall making sure everyone is comfortable. Despite the different tasks laid down below, volunteers represent a single unit. Sometimes this would mean that as a session chair you will be asked about where the quiet room is, we expect you to help this person out or point them to someone who can.
```

Comment:
```suggestion
As a part of EuroPython volunteers' fleet, you will be the face of the conference. Your prime directive will be to ensure everyone coming to the conference has a great time. We want volunteers to be welcoming, helpful, collaborative, and overall making sure everyone (including yourself) is comfortable. Despite the different tasks laid down below, volunteers represent a single unit. Sometimes this would mean that as a session chair you will be asked about where the quiet room is, we expect you to help this person out or point them to someone who can.
```